test/SparqlExpressionTest.cpp

```diff
@@ -43,19 +60,20 @@ auto testNaryExpression = [](const auto& expected, auto&&... operands) {
   LocalVocab localVocab;
   IdTable table{alloc};
 
+  // Get the size of `operand`: size for a vector, 1 otherwise.
   auto getResultSize = []<typename T>(const T& operand) -> size_t {
     if constexpr (isVectorResult<T>) {
       return operand.size();
     }
     return 1;
   };
-
   auto resultSize = std::max({getResultSize(operands)...});
 
   sparqlExpression::EvaluationContext context{*ad_utility::testing::getQec(),
                                               map, table, alloc, localVocab};
   context._endIndex = resultSize;
 
+  // TODO: explain why we need to clone the operands below.
```

Comment:
We need to explicitly clone, because `VectorWithMemoryLimit` does not have a copy constructor. (deliberately)